rtnetlink/src/link/property_del.rs

```diff
@@ -0,0 +1,65 @@
+use crate::{
+    packet::{
+        nlas::link::{Nla, PropList},
+        LinkMessage,
+        NetlinkMessage,
+        NetlinkPayload,
+        RtnlMessage,
+        NLM_F_ACK,
+        NLM_F_APPEND,
+        NLM_F_CREATE,
+        NLM_F_EXCL,
+        NLM_F_REQUEST,
+    },
+    Error,
+    Handle,
+};
+use futures::stream::StreamExt;
+
+pub struct LinkDelPropRequest {
+    handle: Handle,
+    message: LinkMessage,
+}
+
+impl LinkDelPropRequest {
+    pub(crate) fn new(handle: Handle, index: u32) -> Self {
+        let mut message = LinkMessage::default();
+        message.header.index = index;
+        LinkDelPropRequest { handle, message }
+    }
+
+    /// Execute the request
+    pub async fn execute(self) -> Result<(), Error> {
+        let LinkDelPropRequest {
+            mut handle,
+            message,
+        } = self;
+        let mut req = NetlinkMessage::from(RtnlMessage::DelLinkProp(message));
+        req.header.flags = NLM_F_REQUEST | NLM_F_ACK | NLM_F_EXCL | NLM_F_CREATE | NLM_F_APPEND;
```

Comment:
This delete action need flag `NLM_F_CREATE | NLM_F_APPEND`?

Reply:
No they are useless